index.html

```diff
@@ -5931,7 +5931,8 @@ <h5>Note regarding the ARIA 1.1 <code>none</code> role.</h5>
 		<div class="role" id="note">
 			<rdef>note</rdef>
 			<div class="role-description">
-				<p>A section whose content is parenthetic or ancillary to the main content of the resource.</p>
+				<p>A <rref>section</rref> whose content qualifies, explains, annotates or is subordinate to some other piece of content</p>
+				<p>In cases where an element with role <rref>note</rref> has an explicit association with another element, authors SHOULD point to the note from that other element using <pref>aria-details</pref>.</p>
```

Comment:
This introduces a normative change that is arguably subjective.  I would personally prefer that this be changed to an "authors can use..." 

For instance, 
```
<p>Some words about something.</p>
<div role=note><p>supplementary content that supports the previous paragraph...</p></div>
```

authors should not be expected to create an `aria-details` association there.

There could be cases where an `aria-details` association might be helpful... so again, happy to introduce the idea.  just do not want a SHOULD here at all.